<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSelection</class>
 <widget class="QWidget" name="DeviceSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="vl1">
   <item>
    <widget class="QLabel" name="scanLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Scanning for devices -- This can take some time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="selectLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Select scanner to use</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>684</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="scrollLayout">
       <item>
        <spacer name="scrollSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl1">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="text">
        <string>Refresh List</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>refreshBtn</sender>
   <signal>clicked()</signal>
   <receiver>DeviceSelection</receiver>
   <slot>refreshDevices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeviceSelection</sender>
   <signal>finishedReloading()</signal>
   <receiver>DeviceSelection</receiver>
   <slot>devicesLoaded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>finishedReloading()</signal>
  <slot>refreshDevices()</slot>
  <slot>devicesLoaded()</slot>
 </slots>
</ui>
